client/src/AppBundle/Resources/views/Report/ProfDeputyCosts/breakdown.html.twig

```diff
@@ -8,7 +8,7 @@
 {% set page = 'breakdown' %}
 
 {% block htmlTitle %}{{ (page ~ '.htmlTitle') | trans }}{% endblock %}
-{% block pageTitle %}{{ (page ~ '.pageSectionDescription') | trans }}{% endblock %}
+{% block pageTitle %}{{ (page ~ '.pageSectionDescription') | trans(transOptions) }}{% endblock %}
```

Comment:
This is where the bug lied with the `pageTitle` printing `%client%` instead of the actual client's name due to use not replacing the variable with the value we have